pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -123,6 +123,16 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return !utils.Includes(constants.ServiceAnnotationDisallowedList, key)
 	})
 
+	deployConfig, err := v1beta1api.NewDeployConfig(r.Client)
+	if err != nil {
+		return reconcile.Result{}, errors.Wrapf(err, "fails to create DeployConfig")
+	}
+
+	if isvcutils.GetDeploymentMode(annotations, deployConfig) == constants.ModelMeshDeployment {
+		r.Log.Info(string(constants.ModelMeshDeployment), " is set as ", constants.DeploymentMode,
```

Comment:
Log message has mismatched key-value pairs. Suggestion to clean up the message a bit:
```golang
r.Log.Info("Skipping reconciliation for InferenceService", constants.DeploymentMode, deploymentMode,
    "apiVersion", isvc.APIVersion, "isvc", isvc.Name)
```